<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KMoreToolsConfigWidget</class>
 <widget class="QWidget" name="KMoreToolsConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1001</width>
    <height>452</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QGroupBox" name="gbMainSection">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Main section</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QListWidget" name="listMainSection"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>170</width>
         <height>325</height>
        </size>
       </property>
       <widget class="QToolButton" name="buttonMoveDown">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>190</y>
          <width>81</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Move down</string>
        </property>
        <property name="icon">
         <iconset theme="arrow-down"/>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
       </widget>
       <widget class="QToolButton" name="buttonMoveUp">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>140</y>
          <width>81</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Move up</string>
        </property>
        <property name="icon">
         <iconset theme="arrow-up"/>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
       </widget>
       <widget class="QToolButton" name="buttonMoveToMain">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>90</y>
          <width>147</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Move to Main section</string>
        </property>
        <property name="icon">
         <iconset theme="arrow-left"/>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
       </widget>
       <widget class="QToolButton" name="buttonMoveToMore">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>148</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Move to More section</string>
        </property>
        <property name="icon">
         <iconset theme="arrow-right"/>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
       </widget>
       <widget class="QToolButton" name="buttonReset">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>280</y>
          <width>81</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Reset</string>
        </property>
        <property name="icon">
         <iconset theme="document-revert"/>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="gbMoreSection">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>More section</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QListWidget" name="listMoreSection"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QFrame" name="frameNotInstalledTools">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>0</number>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>For this menu there are some more tools suggested which are currently not installed:</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonNotInstalledTools">
          <property name="text">
           <string>More tools...</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
